systemtest/src/test/java/io/strimzi/systemtest/security/custom/CustomAuthorizerST.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.security.custom;
+
+
+import io.strimzi.systemtest.AbstractST;
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.kafkaclients.internalClients.InternalKafkaClient;
+import io.strimzi.systemtest.resources.ResourceManager;
+import io.strimzi.systemtest.resources.crd.KafkaClientsResource;
+import io.strimzi.systemtest.resources.crd.KafkaResource;
+import io.strimzi.systemtest.resources.crd.KafkaTopicResource;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import kafka.security.authorizer.AclAuthorizer;
+
+import static io.strimzi.systemtest.Constants.INTERNAL_CLIENTS_USED;
+import static io.strimzi.test.k8s.KubeClusterResource.kubeClient;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@Tag(INTERNAL_CLIENTS_USED)
+public class CustomAuthorizerST extends AbstractST {
+    static final String NAMESPACE = "custom-authorizer-test";
+    static final String CLUSTER_NAME = "custom-authorizer";
+    static final String ADMIN = "sre-admin";
+
+    @BeforeAll
```

Comment:
Please, move `beforeAll` to the bottom of the class as we have it in all other classes.